cl/users/views.py

```diff
@@ -289,13 +293,24 @@ def view_settings(request: AuthenticatedHttpRequest) -> HttpResponse:
         user_form.save()
 
         return HttpResponseRedirect(reverse("view_settings"))
+
+    email_broken = broken_email_address(user)
+    if email_broken["EMAIL_BROKEN_BANNER"]["show"]:
+        email_error_message = determine_email_error_message(
+            email_broken["EMAIL_BROKEN_BANNER"]["type"],
+            email_broken["EMAIL_BROKEN_BANNER"]["sub_type"],
+        )
+    else:
+        email_error_message = ""
+
```

Comment:
Just to follow up on our conversation this should be refactored for simplicity.

Reply:
Done, since we don't need the `determine_email_error_message` anymore, I just keep the required code here.